public/partials/buyer_profile_large.html

```diff
@@ -146,9 +152,70 @@ <h3 class="{{new_demand? 'dark':'white'}}">Demands</h3>
                     <div ng-include="new_demand ? '/partials/demand_new.html':null">
 
                     </div>
+
+                    <div ng-include="edit_demand ? '/partials/demand_edit.html':null">
+
+                    </div>
+
                 </md-card-content>
-                <md-card-content>
-                    <div class="farmerInfo" layout-padding layout="column" ng-repeat="demand in demands">
+                <md-card-content ng-show="!edit_demand">
+
+
+                    <style>
```

Comment:
No way... stop mixing up the HTML with CSS.  This is EXTREMELY difficult to maintain.  This is sloppy.